### PR TITLE
🌟feat: installation

### PR DESCRIPTION
**Pull request by nandiheath**
initial version for getting started

### CODE REVIEW

**Review by sumitkolhe — APPROVED** [review]

File: docusaurus.config.js [hunk 1]
```diff
@@ -95,11 +95,6 @@ module.exports = {
 
       copyright: `<p style="text-align:left;">Open-sourced with Apache 2.0 License.</p>`,
     },
-    algolia: {
```
Comment (sumitkolhe) [inline docusaurus.config.js]: Why removed? dont we need it for search?

File: docs/getting-started/introduction.md [hunk 2]
```diff
@@ -4,45 +4,20 @@ title: Introduction
...
+KintoHub is an open source build and deployment platform designed with a developer-friendly interface for Kubernetes.
 
-**KintoHub** makes it easy to manage, debug and scale all-of-the above through an intuitive Dashboard across one or many regions.
+- Build your code from git repository and deploy it on Kubernetes
+- Provide CLI/Dashboard for accessing your applications deployed
+- Generate TLS certificates automatically for supporting HTTPS for your applications    
+- Fullstack - you can deploy websites/APIs/jobs and even databases
 
-## KintoHub Mission
+## Why Kintohub
```
Comment (sumitkolhe) [inline docs/getting-started/introduction.md]: maybe add a question mark at the end - > Why KintoHub?

File: docs/getting-started/introduction.md [hunk 3]
```diff
@@ -4,45 +4,20 @@ title: Introduction
 
 ---
 
-If your app or idea requires:
+## What is Kintohub
```
Comment (sumitkolhe) [inline docs/getting-started/introduction.md]: add a question mark at the end

File: docusaurus.config.js (hunk 1 above)
Comment (nandiheath) [reply docusaurus.config.js]: good catch! I was doing some tests and which I shouldn't commit this line

**Pull request merged**